charts/seed-controlplane/charts/csi-packet/templates/controller.yaml

```
@@ -0,0 +1,115 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-packet-pd
+  labels:
+    app: csi-packet-pd
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  selector:
+    app: csi-packet-pd
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+
+kind: StatefulSet
+apiVersion: {{ include "statefulsetversion" . }}
+metadata:
+  name: csi-packet-controller
+  namespace: kube-system
```

```suggestion
  namespace: {{ .Release.Namespace }}
```

This should be deployed in the shoot namespace in the seed (`shoot--foo--bar`)

---

Oh yes. Which is just interpolated. Fixing.

---

Resolved (and everywhere else in this chart)